src/test/java/org/broadinstitute/hellbender/tools/walkers/genotyper/AlleleSubsettingUtilsUnitTest.java

```diff
@@ -130,9 +133,9 @@ public void testUpdatePLsAndADData(final VariantContext originalVC,
                 Collections.singletonList(new GenotypeBuilder(base).alleles(Arrays.asList(Aref, Aref, C)).PL(new double[]{-10, 0, -20, -30}).AD(new int[]{10, 10}).GQ(100).make())});
```

Comment:
Same here -- I'd rather a 2:1 ratio on AD.  And the PLs don't look big enough.

Reply:
done and made the PLs more realistic